### PR TITLE
Overrides window.open for background and extension page

### PR DESCRIPTION
**Pull request by darkdh**
fix https://github.com/brave/browser-laptop/issues/4686

@Auditors: @bridiver, @jonathansampson

### CODE REVIEW

**Comment by jonathansampson** [comment]
Looks good to me! @bridiver, safe to merge?

File: atom/common/api/resources/ipc_bindings.js [hunk 1]
```diff
@@ -5,19 +5,24 @@ exports.didCreateDocumentElement = function() {
...
   }
 
   window.prompt = function(text, defaultText) {
-    return ipc.sendSync('window-prompt', text, defaultText);
+    return ipc.sendSync('window-prompt', text, defaultText)
   }
+
+  if (window.chrome && window.chrome.tabs && window.chrome.tabs.create)
+    window.open = function(url, name, specs, replace) {
+      return window.chrome.tabs.create({url: url})
```
Comment (bridiver) [inline atom/common/api/resources/ipc_bindings.js]: not handling the other args is ok for now, but we should at least throw an error if anything other than url is passed

File: atom/common/api/resources/ipc_bindings.js [hunk 2]
```diff
@@ -5,19 +5,24 @@ exports.didCreateDocumentElement = function() {
   if (!window)
     return
 
-  var slice = [].slice;
+  var slice = [].slice
```
Comment (bridiver) [inline atom/common/api/resources/ipc_bindings.js]: yea, I think it's leftover from when this actually had ipc related code in it

**Comment by jonathansampson** [comment]
I'm a little nervous about throwing, but I haven't yet encountered an extension that issues more than a URL.

**Pull request merged**